woocommerce/woocommerce
# Customize Your Store - Homepage sidebar

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #39720 .



### How to test the changes in this Pull Request:



Note: Certain patterns are exclusive to WPCOM. Therefore, when conducting tests, utilizing a DEV WooExpress site is advisable, although it is acceptable if you don't. Your JN or local site will only show the supported patterns.

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Use WP 6.3 
2. Install the `WooCommerce Beta Tester` plugin
3. Go to /wp-admin/tools.php?page=woocommerce-admin-test-helper and enable `customize-store` feature flag
4. Go to `WooCommerce > Home`
5. Click on `Customize your store` task 
6. Click on `Assembler Hub` button
7. Click on `Change your homepage`
8. Confirm homepage patterns are loaded on the sidebar
9. Click on one of the patterns to confirm that it replaces the preview blocks.

https://github.com/woocommerce/woocommerce/assets/4344253/c05d0e34-8a7e-4519-bfa3-510b5d8db8c8




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Add `Change your homepage` component to Customize Your Store task.

#### Comment 

</details>
